[2024-12-06T21:28:06.067+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T21:28:06.104+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: homework_zone_dag.local_to_gcs manual__2024-12-06T21:27:55.272504+00:00 [queued]>
[2024-12-06T21:28:06.122+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: homework_zone_dag.local_to_gcs manual__2024-12-06T21:27:55.272504+00:00 [queued]>
[2024-12-06T21:28:06.122+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-06T21:28:06.158+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): local_to_gcs> on 2024-12-06 21:27:55.272504+00:00
[2024-12-06T21:28:06.176+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5238) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T21:28:06.180+0000] {standard_task_runner.py:72} INFO - Started process 5239 to run task
[2024-12-06T21:28:06.179+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'homework_zone_dag', 'local_to_gcs', 'manual__2024-12-06T21:27:55.272504+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/homework_zone_dag.py', '--cfg-path', '/tmp/tmp51islbt8']
[2024-12-06T21:28:06.185+0000] {standard_task_runner.py:105} INFO - Job 145: Subtask local_to_gcs
[2024-12-06T21:28:06.264+0000] {task_command.py:467} INFO - Running <TaskInstance: homework_zone_dag.local_to_gcs manual__2024-12-06T21:27:55.272504+00:00 [running]> on host 39ecaeffba2c
[2024-12-06T21:28:06.388+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='homework_zone_dag' AIRFLOW_CTX_TASK_ID='local_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T21:27:55.272504+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T21:27:55.272504+00:00'
[2024-12-06T21:28:06.393+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T21:28:07.179+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T21:28:07.193+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T21:28:07.193+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=homework_zone_dag, task_id=local_to_gcs, run_id=manual__2024-12-06T21:27:55.272504+00:00, execution_date=20241206T212755, start_date=20241206T212806, end_date=20241206T212807
[2024-12-06T21:28:07.257+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T21:28:07.281+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-06T21:28:07.282+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
